Felienne/hedy
# [BUG FIX] Catches error on translating keywords

**Description**
In this PR we fix an issue where the page would return an unclear error when translating a code snippet that for some reason did not translate successfully. We add a try/catch and return an nice error message instead.

**Fixes**
This PR fixes #2050 

**How to test**
Navigate to level 4, select the "rock, paper, scissor" adventure and try to translate. This should no longer result in a page crash and return a nice "something went wrong when translating" error.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
